Tests/SwiftSyntaxMacroExpansionTest/AccessorMacroTests.swift

```diff
@@ -44,6 +44,95 @@ fileprivate struct ConstantOneGetter: AccessorMacro {
 final class AccessorMacroTests: XCTestCase {
   private let indentationWidth: Trivia = .spaces(2)
 
+  func testAccessorOnVariableDeclWithTrailingLineCommentAndNoAccessorBlock() {
+    assertMacroExpansion(
+      """
+      @constantOne
+      var x: Int // hello
+      """,
+      expandedSource: """
+        var x: Int /* hello*/ {
```

Comment:
I was bikeshedding between keeping the style intact or converting them into block, I picked the conversion route because it was slightly easier to code ;)

I agree source fidelity is important, I'll make some changes in the next commit.

Reply:
@bnbarham, please have a look at my latest commit.